### PR TITLE
For tdvirsh new, switch from environment vars to command parameters

### PR DESCRIPTION
**Pull request by BFuhry**

### CODE REVIEW

**Review by hector-cao — APPROVED** [review]
Hello @BFuhry ; LGTM  and thanks

**Review by hector-cao — CHANGES_REQUESTED** [review]
Can you update also the README please ? at https://github.com/canonical/tdx/blob/noble-24.04/README.md#63-secure-boot-td

**Comment by BFuhry** [comment]
> Can you update also the README please ? at https://github.com/canonical/tdx/blob/noble-24.04/README.md#63-secure-boot-td

Done

File: README.md [hunk 1]
```diff
@@ -235,11 +237,11 @@ A different qcow2 image (e.g., one with an `intel` kernel) can be used by settin
 
 ### 6.3 Secure Boot TD
 
-To run a TD VM with secure boot enabled, use the `tdvirsh` tool with
+To run a TD with secure boot enabled, use the `tdvirsh` tool with
 dedicated libvirt configuration:
 
 	cd tdx/guest-tools
-	XML_TEMPLATE=trust_domain-sb.xml.template ./tdvirsh new
+	./tdvirsh new -i trust_domain-sb.xml.template
```
Comment (hector-cao) [inline README.md:244]: Is that rather `-t` instead of `-i`

**Review by hector-cao — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BFuhry) [reply README.md]: Yep, sorry for that. Fixed in new commit. 
Please squash all my commits on merge using the commit message only from the first commit.

**Review by hector-cao — APPROVED** [review]
Thanks

**Pull request merged**